### PR TITLE
modified travis.yml to cache gradle files

### PR DESCRIPTION
**Pull request by dynamitechetan**
Fixes issue #92

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +30,19 @@ android:
 
     # Specify at least one system image
     - sys-img-armeabi-v7a-google_apis-$ANDROID_API_LEVEL
+    
+# prevents reuploading of Cache
+before_cache:
+  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+
+# list of directoriese to Cahce
```
Comment (Abhi2424shek) [inline .travis.yml]: Typo in line. (Cahce)

**Comment by Abhi2424shek** [comment]
Next time squash your commits, I squashed it this time.

**Comment by dynamitechetan** [comment]
okay!

**Comment by Abhi2424shek** [comment]
Merging!

**Pull request merged**